tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -524,6 +524,82 @@ async def test_inspector_prompt_and_stats_render_host_diagnostics(
         )
         await expect(stats.get_by_text("mock-model", exact=True)).to_be_visible()
         await expect(stats.get_by_text("Statistics are partial:")).to_have_count(0)
+
+        second_marker = f"turn-navigation-e2e-{uuid.uuid4()}"
+        async with httpx.AsyncClient(headers=headers) as client:
```

Ruff (0.16.1)

[error] Undefined name `headers`

(F821)